<launch>
    <node name="ik_service" type="ik_service" pkg="ik_service" output="screen" />
</launch>